CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v0.120.4, 1 October 2020
+
+- Go: Bump golang to v1.15.2 (thanks, @chenrui333!)
+- Bump phpstan/phpstan from 0.12.45 to 0.12.47 in /composer/helpers
+- Upgrade Python to 3.8.6 (@ulgens)
+- Handle empty pipfile requirement string
+- Teach FileFetcher to fetch from disk if local repository clone is present
```

Comment:
The diff from v0.120.3 to the latest code on main (https://github.com/dependabot/dependabot-core/compare/v0.120.3...5682a055e36de810d9af3b8a353cd0a1a1e1e3d6) also includes these 3 PRs, but I _think_ they're only relevant to people that are actively developing in the dependabot-core codebase (as opposed to people using dependabot-core), so I've excluded them from the CHANGELOG:

- #2577
- #2579
- #2584

If any of these should be in the CHANGELOG, just let me know. :-)

Reply:
The first two seem internal, 💯 .

The third affects the production `bundler` update path, so I think it's worth a mention.
It _should_ be a no-op refactor, but it'd be a bummer if someone's bundler broke because of an unmentioned change.